string receptacle
home_robot_msgs/RGBDImage image
---
geometry_msgs/Pose2D goal_pose